crates/biome_json_syntax/src/file_source.rs

```diff
@@ -106,7 +89,7 @@ fn compute_source_type_from_path_or_extension(
     } else {
         match extension {
             "json" => JsonFileSource::json(),
-            "jsonc" => JsonFileSource::jsonc(),
+            "jsonc" => JsonFileSource::json_with_comments_and_trailing_commas(),
```

Comment:
I'm not sure we should enable trailing commas support for JSONC files. This could be a breaking change.

Reply:
I don't know where I get the faulty impression that trailing commas are allowed in a `.jsonc` file 😓. I can change this to avoid introducing breaking changes.

Reply:
Probably VSCode! 😄